README.md

```diff
@@ -186,6 +186,32 @@ getFID(sendToGoogleAnalytics);
 getLCP(sendToGoogleAnalytics);
 ```
 
+### Send the results to Google Tag Manager
+
+The following example measures each of the Core Web Vitals metrics and sends them as seperate dataLayer-events to be used by Google Tag Manager. With the `web-vitals` trigger you send the metrics to any tag inside your account.
+
+```js
+import {getCLS, getFID, getLCP} from 'web-vitals';
+
+function sendToGoogleAnalytics({name, delta, id}) {
+  // Assumes the global `dataLayer` array exists, see:
+  // https://developers.google.com/tag-manager/devguide
+  dataLayer.push({
+      event: 'web-vitals',
+      event_category: 'Web Vitals',
+      // Google Analytics metrics must be integers, so the value is rounded.
+      // For CLS the value is first multiplied by 1000 for greater precision
+      // (note: increase the multiplier for greater precision if needed).
+      event_value: Math.round(name === 'CLS' ? delta * 1000 : delta),
```

Comment:
Yeah, I found myself multiply by 100 in my codebase to have something like a score from 0 to 100. Have you seen many results with three decimals points?

Reply:
I havn't production tested it yet, so not sure. So I took this x1000 approach because it was suggested in both analytics.js and gtag.js sections of the Readme

Reply:
And at least when running the test-servers part of the repo I get number in the 0.00x range for CLS. 100x might be a better find in the real world environments - I have to test that out.

Reply:
Yeah, I chose 1000x for the Google Analytics examples because I wanted to have enough granularity for several CLS beacons to be sent for the same page without introducing too much of a rounding error. If someone were to only do 100x, then the recommended threshold would be <=10, and there are only 9 integer values between 0 and 10 (which didn't feel like enough).

Also, I didn't want to give the impression that CLS was a percentage (we've have confusion around that before).